diff --git a/apps/website/src/components/base/navbar/Navbar.astro b/apps/website/src/components/base/navbar/Navbar.astro
index 2fdc69a..efaccef 100644
--- a/apps/website/src/components/base/navbar/Navbar.astro
+++ b/apps/website/src/components/base/navbar/Navbar.astro
@@ -3,17 +3,19 @@ import type { HTMLAttributes } from 'astro/types';
 import config from '@utils/site-info';
 import NavbarElement from '../navbar/NavbarElement.astro';
 
-interface Props extends HTMLAttributes<'div'> {}
+interface Props extends HTMLAttributes<'div'> {
+	dark?: boolean
+}
 const props = Astro.props;
 ---
 
 <div {...props} class="absolute z-[9999] h-screen max-h-[110px] w-full flex flex-row justify-center px-3 text-[16px]">
 	<nav class="max-w-[1024px] w-full flex flex-col items-center justify-between md:flex-row">
 		<ul class="flex flex-row justify-start gap-4 max-md:mt-4">
-			{config.navbar.left.map((element, index) => <NavbarElement {element} {index} />)}
+			{config.navbar.left.map((element, index) => <NavbarElement {element} {index} dark={props.dark} />)}
 		</ul>
 		<ul class="flex flex-row flex-wrap justify-center gap-4 md:justify-end">
-			{config.navbar.right.map((element, index) => <NavbarElement {element} {index} />)}
+			{config.navbar.right.map((element, index) => <NavbarElement {element} {index} dark={props.dark} />)}
 		</ul>
 	</nav>
 </div>
diff --git a/apps/website/src/components/base/navbar/NavbarElement.astro b/apps/website/src/components/base/navbar/NavbarElement.astro
index 5752522..e3a8c83 100644
--- a/apps/website/src/components/base/navbar/NavbarElement.astro
+++ b/apps/website/src/components/base/navbar/NavbarElement.astro
@@ -9,13 +9,16 @@ import Tag from '../Tag.astro';
 interface Props {
 	element: NavbarElement;
 	index: number;
+	dark?: boolean;
 }
 
-const { element, index } = Astro.props;
+const { element, index, dark } = Astro.props;
 
 function isCurrentPage(url: string): boolean {
 	return Astro.url.pathname === url;
 }
+
+const color = dark ? 'text-dark-secondary' : 'text-gray-700'
 ---
 
 <li class="flex flex-row items-center justify-center text-center sm:relative max-sm:overflow-hidden">
@@ -24,7 +27,7 @@ function isCurrentPage(url: string): boolean {
 ? (
 			<a
 				aria-label="polyfrost logo"
-				class={`p-2 flex flex-row justify-center items-center hover:text-blue-500 ${isCurrentPage(element.path) ? 'text-blue-400' : 'text-gray-700'}`}
+				class={`p-2 flex flex-row justify-center items-center hover:text-blue-500 ${isCurrentPage(element.path) ? 'text-blue-400' : color}`}
 				href={element.path}
 			>
 				{element.text && element.text}
@@ -37,7 +40,7 @@ function isCurrentPage(url: string): boolean {
 				{element.dropdown && (
 					<>
 						<p
-							class={`p-2 flex flex-row justify-center items-center hover:text-blue-500 cursor-default ${isCurrentPage(element.dropdown[0].path ?? '') ? 'text-blue-400' : 'text-gray-700'}`}
+							class={`p-2 flex flex-row justify-center items-center hover:text-blue-500 cursor-default ${element.dropdown.filter(element=>isCurrentPage(element.path ?? '')).length > 0  ? 'text-blue-400' : color}`}
 						>
 							{element.text && element.text}
 							{element.logo && <Icon height={element.logo[2] < 0 ? 'auto' : element.logo[2]} name={element.logo[0]} width={element.logo[1] < 0 ? 'auto' : element.logo[1]} />}
diff --git a/apps/website/src/content/blog/first.md b/apps/website/src/content/blog/first.md
index 2b5f64a..3747488 100644
--- a/apps/website/src/content/blog/first.md
+++ b/apps/website/src/content/blog/first.md
@@ -1,8 +1,8 @@
 ---
-title: 'First post'
-description: 'Lorem ipsum dolor sit amet'
-publishDate: 'Jul 08 2022'
-heroImage: '/blog-placeholder-3.jpg'
+title: "First post"
+description: "Lorem ipsum dolor sit amet"
+publishDate: "Jul 08 2022"
+coverImage: "https://picsum.photos/900/300?random=1"
 ---
 
 Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Vitae ultricies leo integer malesuada nunc vel risus commodo viverra. Adipiscing enim eu turpis egestas pretium. Euismod elementum nisi quis eleifend quam adipiscing. In hac habitasse platea dictumst vestibulum. Sagittis purus sit amet volutpat. Netus et malesuada fames ac turpis egestas. Eget magna fermentum iaculis eu non diam phasellus vestibulum lorem. Varius sit amet mattis vulputate enim. Habitasse platea dictumst quisque sagittis. Integer quis auctor elit sed vulputate mi. Dictumst quisque sagittis purus sit amet.
diff --git a/apps/website/src/content/blog/second.mdx b/apps/website/src/content/blog/second.mdx
new file mode 100644
index 0000000..c453494
--- /dev/null
+++ b/apps/website/src/content/blog/second.mdx
@@ -0,0 +1,31 @@
+---
+title: "How to make a second post, containing a title much longer than its predecessor"
+description: "Lorem ipsum dolor sit amet"
+publishDate: "Jul 04 2025"
+updatedDate: "Jul 05 2025"
+coverImage: "https://picsum.photos/1200/800?random=2"
+---
+
+import Button from "../../components/base/Button.astro";
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Vitae ultricies leo integer malesuada nunc vel risus commodo viverra. Adipiscing enim eu turpis egestas pretium. Euismod elementum nisi quis eleifend quam adipiscing. In hac habitasse platea dictumst vestibulum. Sagittis purus sit amet volutpat. Netus et malesuada fames ac turpis egestas. Eget magna fermentum iaculis eu non diam phasellus vestibulum lorem. Varius sit amet mattis vulputate enim. Habitasse platea dictumst quisque sagittis. Integer quis auctor elit sed vulputate mi. Dictumst quisque sagittis purus sit amet.
+
+![A beautiful image](https://picsum.photos/1200/800?random=3)
+
+> "Mattis ullamcorper velit sed ullamcorper morbi tincidunt. Tortor posuere ac ut consequat semper viverra. Tellus mauris a diam maecenas sed enim ut sem viverra. Venenatis urna cursus eget nunc scelerisque viverra mauris in. Arcu ac tortor dignissim convallis aenean et tortor at. Curabitur gravida arcu ac tortor dignissim convallis aenean et tortor. Egestas tellus rutrum tellus pellentesque eu. Fusce ut placerat orci nulla pellentesque dignissim enim sit amet. Ut enim blandit volutpat maecenas volutpat blandit aliquam etiam. Id donec ultrices tincidunt arcu. Id cursus metus aliquam eleifend mi."
+>
+> -   Socrates, Probably
+
+```ts
+console.log("Hello, World");
+```
+
+Inline code is displayed `like this`... Very interesting.
+
+# Headers
+
+## Headers
+
+<Button text="I am a button!" />
+
+Tempus quam pellentesque nec nam aliquam sem. Risus at ultrices mi tempus imperdiet. Id porta nibh venenatis cras sed felis eget velit. Ipsum a arcu cursus vitae. Facilisis magna etiam tempor orci eu lobortis elementum. Tincidunt dui ut ornare lectus sit. Quisque non tellus orci ac. Blandit libero volutpat sed cras. Nec tincidunt praesent semper feugiat nibh sed pulvinar proin gravida. Egestas integer eget aliquet nibh praesent tristique magna.
diff --git a/apps/website/src/layouts/BlogPost.astro b/apps/website/src/layouts/BlogPost.astro
index a8beef2..0874688 100644
--- a/apps/website/src/layouts/BlogPost.astro
+++ b/apps/website/src/layouts/BlogPost.astro
@@ -2,34 +2,38 @@
 import type { CollectionEntry } from 'astro:content';
 import FormattedDate from '@components/base/FormattedDate.astro';
 import Layout from './Layout.astro';
-import '@styles/blog.css';
 
 type Props = CollectionEntry<'blog'>['data'];
 
 const { title, description, publishDate, updatedDate, coverImage } = Astro.props;
 ---
 
-<Layout description={description} title={title}>
-	<article>
-		<div class="hero-image">
-			{coverImage && <img alt="Hero Image" height={510} src={coverImage} width={1020} />}
-		</div>
-		<div class="prose">
-			<div class="title">
-				<div class="date">
-					<FormattedDate date={publishDate} />
-					{
-						updatedDate && (
-							<div class="last-updated-on">
-								Last updated on <FormattedDate date={updatedDate} />
-							</div>
-						)
-					}
+<div class="parallax-container overflow-x-clip overflow-y-auto max-h-screen">
+	<div class="flex justify-center parallax-3 absolute w-full mt-80 md:mt-24">
+		{coverImage && <img alt="Hero Image" class="object-cover  object-c rounded-t-2xl md:rounded-2xl ring-1 ring-gray-400/50 -z-1 aspect-[3/2]" height={510} src={coverImage} width={1020} />}
+	</div>
+	<Layout description={description} title={title}>
+		<article class="mt-80 md:mt-24">
+			<div class="flex items-start flex-col mx-auto mt-32 mb-12 max-w-[calc(65ch+32px)] gap-6 parallax-0">
+				<div class="w-full bg-gray-50 p-4 ring-1 ring-gray-400/50 rounded-3xl">
+					<div class="text-xs font-light italic">
+						<FormattedDate date={publishDate} />
+						{
+							updatedDate && (<span class="">- Last updated on <FormattedDate date={updatedDate} /></span>)
+
+						}
+					</div>
+					<h1 class="text-3xl font-bold text-navy-peony mb-1">{title}</h1>
+					<p class="text-sm text-gray-700">
+						{description}
+					</p>
+				</div>
+				<div class="bg-gray-50 p-4 ring-1 ring-gray-400/50 rounded-3xl">
+					<article class="prose" >
+						<slot/>
+					</article>
 				</div>
-				<h1>{title}</h1>
-				<hr />
 			</div>
-			<slot />
-		</div>
-	</article>
-</Layout>
+		</article>
+	</Layout>
+</div>
diff --git a/apps/website/src/layouts/Layout.astro b/apps/website/src/layouts/Layout.astro
index 29aa28b..ec32b16 100644
--- a/apps/website/src/layouts/Layout.astro
+++ b/apps/website/src/layouts/Layout.astro
@@ -4,9 +4,11 @@ import Navbar from '../components/base/navbar/Navbar.astro';
 import BaseHead, { type Props as HeadProps } from '../components/shared/BaseHead.astro';
 import '../styles/global.css';
 
-export type Props = HeadProps;
+export interface Props extends HeadProps{
+	dark?:boolean
+};
 
-const { ...head } = Astro.props;
+const { dark, ...head } = Astro.props;
 ---
 
 <!doctype html>
@@ -16,7 +18,7 @@ const { ...head } = Astro.props;
 	</head>
 
 	<body class="overflow-x-hidden bg-gray-50">
-		<Navbar />
+		<Navbar {dark} />
 
 		<main class="h-auto min-h-screen flex flex-col gap-40">
 			<slot />
diff --git a/apps/website/src/pages/blog/index.astro b/apps/website/src/pages/blog/index.astro
index edb8329..e808c5f 100644
--- a/apps/website/src/pages/blog/index.astro
+++ b/apps/website/src/pages/blog/index.astro
@@ -2,26 +2,34 @@
 import { getCollection } from 'astro:content';
 import FormattedDate from '../../components/base/FormattedDate.astro';
 import Layout from '../../layouts/Layout.astro';
+import Section from '@components/base/Section.astro';
 
 const posts = await getCollection('blog');
 ---
 
 <Layout description="Recieve Polyfrost updates here" title="Polyfrost Blog">
-	<section>
-		<ul>
+	<Section class="mt-96 md:mt-32 mb-12">
+		<div class="flex flex-col gap-5 content-center mx-auto">
 			{
-				posts.map((post: { slug: any; data: { coverImage: string | null | undefined; title: unknown; publishDate: Date; }; }) => (
-					<li>
-						<a href={`blog/${post.slug}/`}>
-							<img alt="" height={360} src={post.data.coverImage} width={720} />
-							<h4 class="title">{post.data.title}</h4>
-							<p class="date">
-								<FormattedDate date={post.data.publishDate} />
-							</p>
-						</a>
-					</li>
+				posts.toSorted((p1,p2)=>p2.data.publishDate.valueOf()-p1.data.publishDate.valueOf()).map((post) => (
+					<a href={`/blog/${post.slug}/`}>
+						<div class="flex gap-2 flex-col md:flex-row">
+							<img alt="" height={225} src={post.data.coverImage} width={400} class="w-full md:w-100 rounded-xl ring-1 ring-gray-400/50 bg-gray-400/40 object-cover aspect-video" />
+							<div class="p-1">
+								<p class="date text-xs font-light text-nowrap">
+									<FormattedDate date={post.data.publishDate} />
+								</p>
+								<div class="flex justify-between">
+									<h3 class="text-lg font-semibold text-navy-peony">{post.data.title}</h3>
+								</div>
+								<p class="text-sm font-light">
+									{post.data.description}
+								</p>
+							</div>
+						</div>
+					</a>
 				))
 			}
-		</ul>
-	</section>
+		</div>
+	</Section>
 </Layout>
diff --git a/apps/website/src/pages/projects/onelauncher/index.astro b/apps/website/src/pages/projects/onelauncher/index.astro
new file mode 100644
index 0000000..0f7192e
--- /dev/null
+++ b/apps/website/src/pages/projects/onelauncher/index.astro
@@ -0,0 +1,75 @@
+---
+import Button from "@components/base/Button.astro";
+import Header from "@components/base/Header.astro";
+import Section from "@components/base/Section.astro";
+import Dropdown from "@components/base/Dropdown.astro";
+import Layout from "@layouts/Layout.astro";
+import DownloadVersion from "@components/base/DownloadVersion.astro";
+import { ostring } from "astro:schema";
+
+export const prerender = false
+
+type OS = "Windows"|"MacOS"|"Linux"
+type Architecture = "x86"|"Arm"
+
+type Platform = `${OS} ${Architecture}`
+
+function getPlatform():Platform|undefined{
+	const agent = Astro.request.headers.get("User-Agent")
+
+	const osString = (agent?.match(/\(([^\)]*)\)/) ?? ["",""])[1];
+	if(osString.startsWith("Windows")){
+		return "Windows x86"
+	}else if(osString.startsWith("Macintosh")){
+		return "MacOS Arm"
+	}else if(osString.startsWith("X11") || osString.startsWith("Linux")){
+		return "Linux x86"
+	}
+}
+
+
+
+const versions: {platform: Platform, url: string}[] = [
+	{
+		platform: "Windows x86",
+		url: "/"
+	},
+	{
+		platform: "Linux x86",
+		url: "/"
+	},
+	{
+		platform: "MacOS Arm",
+		url: "/"
+	},
+]
+
+const platform = getPlatform()
+const version = versions.find(version=>version.platform == platform)
+
+---
+
+<div class="w-screen h-screen absolute bg-dark-background -z-1"/>
+<Layout title="OneLauncher" dark>
+	<Section class="h-screen flex-col items-center justify-center md:min-h-[600px] mt-12 md:mt-0 mb-12">
+		<video src="" class="w-full max-w-2xl bg-gray-50"></video>
+		<div class="flex flex-col items-center justify-center gap-y-4">
+			<Header align="center" class="max-w-[600px] text-dark-primary" size="lg">
+				A modern launcher, for a modern world.
+			</Header>
+			<div class="flex flex-row flex-wrap items-center justify-center gap-2">
+				<Dropdown iconLeft="download" text={version ? `Download for ${platform}` :"Download"} href={version?.url}>
+					<div class="flex flex-col gap-1">
+						<div class="bg-dark-background border border-gray-700 text-dark-secondary rounded-xl p-2 gap-1 flex flex-col">
+							{versions.map(version=>
+								<DownloadVersion name={version.platform} url={version.url}/>
+							)}
+						</div>
+					</div>
+				</Dropdown>
+				<Button href="" iconLeft="book-open" style="secondary" text="FAQ" />
+				<Button href="https://polyfrost.org/discord" iconLeft="user" style="secondary" text="Support" />
+			</div>
+		</div>
+	</Section>
+</Layout>
diff --git a/apps/website/src/styles/blog.css b/apps/website/src/styles/blog.css
deleted file mode 100644
index 9cfe3fa..0000000
--- a/apps/website/src/styles/blog.css
+++ /dev/null
@@ -1,37 +0,0 @@
-main {
-	width: calc(100% - 2em);
-	max-width: 100%;
-	margin: 0;
-}
-.hero-image {
-	width: 100%;
-}
-.hero-image img {
-	display: block;
-	margin: 0 auto;
-	border-radius: 12px;
-	box-shadow: var(--box-shadow);
-}
-.prose {
-	width: 720px;
-	max-width: calc(100% - 2em);
-	margin: auto;
-	padding: 1em;
-	color: rgb(var(--gray-dark));
-}
-.title {
-	margin-bottom: 1em;
-	padding: 1em 0;
-	text-align: center;
-	line-height: 1;
-}
-.title h1 {
-	margin: 0 0 0.5em 0;
-}
-.date {
-	margin-bottom: 0.5em;
-	color: rgb(var(--gray));
-}
-.last-updated-on {
-	font-style: italic;
-}
diff --git a/apps/website/src/utils/site-info.ts b/apps/website/src/utils/site-info.ts
index 3471d14..304bb12 100644
--- a/apps/website/src/utils/site-info.ts
+++ b/apps/website/src/utils/site-info.ts
@@ -1,9 +1,9 @@
-import type { Icon } from 'virtual:astro-icon';
+import type { Icon } from "virtual:astro-icon";
 
 export interface ProjectDownload {
 	url: string;
-	platform?: 'windows' | 'mac' | 'linux' | 'universal';
-	architecture?: 'x86' | 'x64' | 'arm' | 'arm64' | 'universal';
+	platform?: "windows" | "mac" | "linux" | "universal";
+	architecture?: "x86" | "x64" | "arm" | "arm64" | "universal";
 }
 
 export interface Project {
@@ -55,7 +55,7 @@ export interface Config {
 		discord: string;
 		github: string;
 		modrinth: {
-			type: 'user' | 'organization';
+			type: "user" | "organization";
 			id: string;
 		};
 		skyclient: string;
@@ -68,107 +68,115 @@ export interface Config {
 }
 
 export const configConst = {
-	name: 'Polyfrost',
-	title: 'Polyfrost',
-	description: 'Building beautiful mods and tools, byte by byte',
+	name: "Polyfrost",
+	title: "Polyfrost",
+	description: "Building beautiful mods and tools, byte by byte",
 	image: {
-		src: '/media/polyfrost/minimal.svg',
-		alt: 'Polyfrost Logo',
+		src: "/media/polyfrost/minimal.svg",
+		alt: "Polyfrost Logo",
 	},
 	projects: getProjects(),
 	socials: {
-		youtube: 'https://youtube.com/@Polyfrost',
-		discord: 'https://discord.gg/N4qW7TW3dv',
-		github: 'https://github.com/Polyfrost',
+		youtube: "https://youtube.com/@Polyfrost",
+		discord: "https://discord.gg/N4qW7TW3dv",
+		github: "https://github.com/Polyfrost",
 		modrinth: {
-			id: 'polyfrost',
-			type: 'organization',
+			id: "polyfrost",
+			type: "organization",
 		},
-		skyclient: 'https://skyclient.co',
+		skyclient: "https://skyclient.co",
 	},
 	navbar: {
 		left: [
 			{
-				logo: ['polyfrost.full', 174, 30],
-				path: '/',
+				logo: ["polyfrost.full", 174, 30],
+				path: "/",
 			},
 		],
 		right: [
 			{
-				text: 'Home',
-				path: '/',
+				text: "Home",
+				path: "/",
 			},
 			{
-				text: 'Mods',
-				path: '/mods',
+				text: "Mods",
+				path: "/mods",
 			},
 			{
-				text: 'Projects',
+				text: "Projects",
 				dropdown: getProjects().map((project) => {
 					if (project.hasPage)
-						(project as any).path = `/projects/${project.name.toLowerCase()}`;
+						(project as any).path =
+							`/projects/${project.name.toLowerCase()}`;
 
 					return project;
 				}) as unknown as NavbarDropdown[],
 			},
 			{
-				text: 'About Us',
-				path: '/about',
+				text: "About Us",
+				path: "/about",
+			},
+			{
+				text: "Blog",
+				path: "/blog",
 			},
 		],
 	},
 	footer: [
 		{
-			header: 'Products',
-			links: getProjects().filter(project => project.hasPage === true).map(project => ({
-				text: project.name,
-				url: `/projects/${project.name.toLowerCase()}`,
-			})).concat([
-				{
-					text: 'Mods',
-					url: '/mods',
-				},
-			]),
+			header: "Products",
+			links: getProjects()
+				.filter((project) => project.hasPage === true)
+				.map((project) => ({
+					text: project.name,
+					url: `/projects/${project.name.toLowerCase()}`,
+				}))
+				.concat([
+					{
+						text: "Mods",
+						url: "/mods",
+					},
+				]),
 		},
 		{
-			header: 'Organization',
+			header: "Organization",
 			links: [
 				{
-					text: 'About us',
-					url: '/about',
+					text: "About us",
+					url: "/about",
 				},
 				{
-					text: 'Branding',
-					url: '/branding',
+					text: "Branding",
+					url: "/branding",
 				},
 				{
-					text: 'Contact us',
-					url: '/contact',
+					text: "Contact us",
+					url: "/contact",
 				},
 				{
-					text: 'Documentation',
-					url: 'https://docs.polyfrost.org',
+					text: "Documentation",
+					url: "https://docs.polyfrost.org",
 				},
 				{
-					text: 'Open source',
-					url: '/oss',
+					text: "Open source",
+					url: "/oss",
 				},
 			],
 		},
 		{
-			header: 'Legal',
+			header: "Legal",
 			links: [
 				{
-					text: 'Terms of service',
-					url: '/legal/terms',
+					text: "Terms of service",
+					url: "/legal/terms",
 				},
 				{
-					text: 'Privacy policy',
-					url: '/legal/privacy',
+					text: "Privacy policy",
+					url: "/legal/privacy",
 				},
 				{
-					text: 'Analytics Opt-out / in',
-					url: '/legal/analytics',
+					text: "Analytics Opt-out / in",
+					url: "/legal/analytics",
 				},
 			],
 		},
@@ -178,16 +186,25 @@ export const configConst = {
 function getProjects(): Project[] {
 	return [
 		{
-			name: 'OneConfig',
-			description: 'The next-generation config library for Forge and Fabric',
-			logo: 'oneconfig.minimal',
+			name: "OneConfig",
+			description:
+				"The next-generation config library for Forge and Fabric",
+			logo: "oneconfig.minimal",
 			hasPage: true,
 		},
 		{
-			name: 'OneLauncher',
-			description: 'The next-generation launcher for all your Minecraft needs',
-			logo: 'onelauncher.minimal',
-			tag: 'SOON',
+			name: "OneLauncher",
+			description:
+				"The next-generation launcher for all your Minecraft needs",
+			logo: "onelauncher.minimal",
+			tag: "SOON",
+		},
+		{
+			name: "OneLauncher",
+			description:
+				"The next-generation launcher for all your Minecraft needs",
+			logo: "onelauncher.minimal",
+			hasPage: true,
 		},
 	];
 }
diff --git a/apps/website/uno.config.ts b/apps/website/uno.config.ts
index d8ef434..6d271cb 100644
--- a/apps/website/uno.config.ts
+++ b/apps/website/uno.config.ts
@@ -1,113 +1,144 @@
-import { defineConfig, presetIcons, presetUno, transformerDirectives, transformerVariantGroup } from 'unocss';
+import {
+	defineConfig,
+	presetIcons,
+	presetTypography,
+	presetUno,
+	transformerDirectives,
+	transformerVariantGroup,
+} from "unocss";
 
 export default defineConfig({
-	rules: [],
+	rules: [
+		["parallax-container", { perspective: "10px" }],
+		[
+			/^parallax-(\d+)$/,
+			([, d]) => ({
+				transform: `translateZ(-${d}px) scale(${parseInt(d) / 10 + 1})`,
+			}),
+		],
+	],
 	shortcuts: {},
 	presets: [
 		presetUno(),
 		presetIcons(),
+		presetTypography({
+			cssExtend: {
+				blockquote: {
+					"border-left": "4px solid rgba(138, 150, 168)",
+					"border-radius": "12px",
+				},
+				img: {
+					"border-radius": "12px",
+				},
+				"p:first-child": {
+					"margin-top": "0",
+				},
+				"h1, h2, h3": {
+					color: "rgba(32, 55, 91)",
+				},
+			},
+		}),
 	],
-	transformers: [
-		transformerVariantGroup(),
-		transformerDirectives(),
-	],
+	transformers: [transformerVariantGroup(), transformerDirectives()],
 	theme: {
 		colors: {
-			'blue': {
-				20: 'rgba(223, 236, 253)',
-				30: 'rgba(183, 208, 251)',
-				50: 'rgba(231, 235, 252)',
-				60: 'rgba(0, 72, 197)',
-				75: 'rgba(227, 236, 245)',
-				100: 'rgba(210, 225, 249)',
-				200: 'rgba(189, 215, 249)',
-				300: 'rgba(166, 200, 249)',
-				400: 'rgba(56, 132, 255)',
-				450: 'rgba(37,99,235)',
-				500: 'rgba(31, 101, 214)',
-				600: 'rgba(9, 84, 165)',
-				800: 'rgba(19, 43, 83)',
+			blue: {
+				20: "rgba(223, 236, 253)",
+				30: "rgba(183, 208, 251)",
+				50: "rgba(231, 235, 252)",
+				60: "rgba(0, 72, 197)",
+				75: "rgba(227, 236, 245)",
+				100: "rgba(210, 225, 249)",
+				200: "rgba(189, 215, 249)",
+				300: "rgba(166, 200, 249)",
+				400: "rgba(56, 132, 255)",
+				450: "rgba(37,99,235)",
+				500: "rgba(31, 101, 214)",
+				600: "rgba(9, 84, 165)",
+				800: "rgba(19, 43, 83)",
 			},
-			'green': {
-				300: 'rgba(35, 154, 96, 0.5)',
+			green: {
+				300: "rgba(35, 154, 96, 0.5)",
 			},
-			'gray': {
-				50: 'rgba(240, 242, 244)',
-				100: 'rgba(196, 202, 212)',
-				200: 'rgba(196, 202, 212)',
-				400: 'rgba(138, 150, 168)',
-				600: 'rgba(42, 44, 48)',
-				700: 'rgba(65, 74, 88)',
-				800: 'rgba(42, 47, 55)',
+			gray: {
+				50: "rgba(240, 242, 244)",
+				100: "rgba(196, 202, 212)",
+				200: "rgba(196, 202, 212)",
+				400: "rgba(138, 150, 168)",
+				600: "rgba(42, 44, 48)",
+				700: "rgba(65, 74, 88)",
+				800: "rgba(42, 47, 55)",
 			},
-			'white': {
-				'DEFAULT': 'rgba(255, 255, 255)',
-				'1/4': 'rgba(255, 255, 255, 0.25)',
-				'light': 'rgba(235, 245, 254)',
+			white: {
+				DEFAULT: "rgba(255, 255, 255)",
+				"1/4": "rgba(255, 255, 255, 0.25)",
+				light: "rgba(235, 245, 254)",
 			},
-			'black': {
-				DEFAULT: 'rgba(0, 0, 0)',
+			black: {
+				DEFAULT: "rgba(0, 0, 0)",
 			},
-			'text': {
-				DEFAULT: 'rgba(2, 3, 7)',
-				primary: 'rgba(2, 3, 7)',
+			text: {
+				DEFAULT: "rgba(2, 3, 7)",
+				primary: "rgba(2, 3, 7)",
 			},
 			// Other
-			'navy-peony': 'rgba(32, 55, 91)',
-			'lightslategray': 'rgba(119, 143, 183)',
-			'primary': {
-				100: 'rgba(23, 28, 33)',
-				200: 'rgba(50, 74, 245)',
-				600: 'rgba(20, 82, 204)',
+			"navy-peony": "rgba(32, 55, 91)",
+			lightslategray: "rgba(119, 143, 183)",
+			primary: {
+				100: "rgba(23, 28, 33)",
+				200: "rgba(50, 74, 245)",
+				600: "rgba(20, 82, 204)",
+			},
+			dark: {
+				background: "rgba(17, 23, 28)",
+				primary: "rgba(213, 219, 255)",
+				secondary: "rgba(120, 129, 141)",
 			},
 		},
 		borderRadius: {
-			'none': '0',
-			'sm': '3px',
-			'md': '5px',
-			'lg': '8px',
-			'xl': '12px',
-			'2xl': '16px',
-			'3xl': '20px',
-			'4xl': '24px',
-			'full': '100vw',
+			none: "0",
+			sm: "3px",
+			md: "5px",
+			lg: "8px",
+			xl: "12px",
+			"2xl": "16px",
+			"3xl": "20px",
+			"4xl": "24px",
+			full: "100vw",
 		},
 		fontSize: {
 			// rem starts at 16px on desktop, 14px on tailwind 'sm' and below
-			'xxs': ['0.625rem', '1rem'], // 10px
-			'xs': ['0.75rem', '1rem'], // 12px
-			'sm': ['0.875rem', 'inherit'], // 14px
-			'md': ['1rem', 'inherit'], // 16px
-			'lg': ['1.125rem', 'inherit'], // 18px
-			'xl': ['1.25rem', 'inherit'], // 20px
+			xxs: ["0.625rem", "1rem"], // 10px
+			xs: ["0.75rem", "1rem"], // 12px
+			sm: ["0.875rem", "inherit"], // 14px
+			md: ["1rem", "inherit"], // 16px
+			lg: ["1.125rem", "inherit"], // 18px
+			xl: ["1.25rem", "inherit"], // 20px
 
-			'header-sm': ['1.5rem', 'inherit'], // 24px
-			'header': ['1.75rem', 'inherit'], // 28px
-			'header-lg': ['2rem', 'inherit'], // 32px
-			'header-page': ['2.25rem', 'inherit'], // 36px
+			"header-sm": ["1.5rem", "inherit"], // 24px
+			header: ["1.75rem", "inherit"], // 28px
+			"header-lg": ["2rem", "inherit"], // 32px
+			"header-page": ["2.25rem", "inherit"], // 36px
 
-			'body-sm': ['0.938rem', 'inherit'], // 15px
-			'body': ['1rem', 'inherit'], // 16px
-			'body-lg': ['1.063rem', 'inherit'], // 17px
+			"body-sm": ["0.938rem", "inherit"], // 15px
+			body: ["1rem", "inherit"], // 16px
+			"body-lg": ["1.063rem", "inherit"], // 17px
 		},
 		fontFamily: {
-			mono: ['"Roboto Mono"', 'monospace'],
+			mono: ['"Roboto Mono"', "monospace"],
 		},
 		extend: {
 			// zIndex: {
 			// 	'navbar': '9999', // Nothing should be above the navbar or backdrop
 			// 	'navbar-backdrop': '9998',
 			// },
-
 			// maxHeight: {
 			// 	'3/4-screen': '75vh',
 			// 	'4/5-screen': '80vh',
 			// },
-
 			// lineHeight: {
 			// 	none: '0',
 			// },
-
 			// transitionProperty: {
 			// 	filter: 'filter',
 			// },
